tox.ini

```diff
@@ -12,7 +12,7 @@ envlist =
 # linters :: Runs all linters over all source code.
 # linters-tests :: Runs all linters over all tests.
 
-ignore_basepython_conflict = True
+# ignore_basepython_conflict = True
```

Comment:
Minor: can we either remove this, or add a comment explaining why I might want to sometimes comment it out?

Reply:
I think we can safely remove it.